### PR TITLE
Event renames - vNext

### PR DESCRIPTION
**Pull request by 3phase**
Related issue: #2920 

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: jp/components/grid/selection-based-aggregates.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ _keywords: データ集計, 選択, ignite ui for angular, インフラジステ
 集計では、列のデータ タイプとニーズに応じて、[`IgxSummaryOperand`]({environment:angularApiUrl}/classes/igxsummaryoperand.html)、[`IgxNumberSummaryOperand`]({environment:angularApiUrl}/classes/igxnumbersummaryoperand.html)、[`IgxDateSummaryOperand`]({environment:angularApiUrl}/classes/igxdatesummaryoperand.html) のいずれかの基本クラスを拡張することにより、基本的な集計機能をカスタマイズできます。  
 
 ## 選択
-選択したグリッド範囲のデータの操作を開始するには、グリッド選択の変更を通知するイベントにサブスクライブする必要があります。これは、[`onSelection`]({environment:angularApiUrl}/classes/igxgridcomponent.html#onrangeselection) と [`onRangeSelection`]({environment:angularApiUrl}/classes/igxgridcomponent.html#onselection) イベントにサブスクライブすることで実行できます。選択機能では、単一のセル選択とセル範囲の選択が区別されるため、両方にバインドする必要があります。     
+選択したグリッド範囲のデータの操作を開始するには、グリッド選択の変更を通知するイベントにサブスクライブする必要があります。これは、[`selected`]({environment:angularApiUrl}/classes/igxgridcomponent.html#rangeSelected) と [`rangeSelected`]({environment:angularApiUrl}/classes/igxgridcomponent.html#selected) イベントにサブスクライブすることで実行できます。選択機能では、単一のセル選択とセル範囲の選択が区別されるため、両方にバインドする必要があります。
```
Comment (mddragnev) [inline jp/components/grid/selection-based-aggregates.md:17]: Same as the en version

File: jp/components/grid/selection-based-aggregates.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ _keywords: データ集計, 選択, ignite ui for angular, インフラジステ
...
 
 ## 選択
-選択したグリッド範囲のデータの操作を開始するには、グリッド選択の変更を通知するイベントにサブスクライブする必要があります。これは、[`onSelection`]({environment:angularApiUrl}/classes/igxgridcomponent.html#onrangeselection) と [`onRangeSelection`]({environment:angularApiUrl}/classes/igxgridcomponent.html#onselection) イベントにサブスクライブすることで実行できます。選択機能では、単一のセル選択とセル範囲の選択が区別されるため、両方にバインドする必要があります。     
+選択したグリッド範囲のデータの操作を開始するには、グリッド選択の変更を通知するイベントにサブスクライブする必要があります。これは、[`selected`]({environment:angularApiUrl}/classes/igxgridcomponent.html#rangeSelected) と [`rangeSelected`]({environment:angularApiUrl}/classes/igxgridcomponent.html#selected) イベントにサブスクライブすることで実行できます。選択機能では、単一のセル選択とセル範囲の選択が区別されるため、両方にバインドする必要があります。     
 
-イベント サブスクリプション ロジックでは、グリッドの [`getSelectedData`]({environment:angularApiUrl}/classes/igxgridcomponent.html#onrangeselection) 関数を使用して選択したデータを抽出し、選択したデータをカスタム集計オペランドに渡すことができます。
+イベント サブスクリプション ロジックでは、グリッドの [`getSelectedData`]({environment:angularApiUrl}/classes/igxgridcomponent.html#rangeSelected) 関数を使用して選択したデータを抽出し、選択したデータをカスタム集計オペランドに渡すことができます。
```
Comment (mddragnev) [inline jp/components/grid/selection-based-aggregates.md:19]: The link should point to `getSelectedData` not `rangeSelected`

File: jp/components/grids_templates/column-moving.md [hunk 3]
```diff
@@ -114,16 +114,16 @@ const idColumn = grid.getColumnByName("ID");
...
+API を使用する時、操作が成功した場合、[`columnMovingEnd`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#columnMovingEnd) イベントのみが発行されることに注意してください。また、ドラッグアンドドロップ機能と比較して、API を使用するために `movable` プロパティを true に設定する必要がないことにも注意してください。 
 
 ## イベント
 
-列のドラッグアンドドロップ操作をカスタマイズするための列移動に関連するイベントが複数あります。[`onColumnMovingStart`]({environment:angularApiUrl}/classes/igxgridcomponent.html#oncolumnmovingstart)、[`onColumnMoving`]({environment:angularApiUrl}/classes/igxgridcomponent.html#oncolumnmoving)、[`onColumnMovingEnd`]({environment:angularApiUrl}/classes/igxgridcomponent.html#oncolumnmovingend) があります。
-[`@@igSelector`]({environment:angularApiUrl}/classes/@@igTypeDoc.html) の [`onColumnMovingEnd`]({environment:angularApiUrl}/classes/igxgridcomponent.html#oncolumnmovingend) イベントを処理し、列が新しい位置にドロップされたときにカスタム ロジックを実装できます。たとえば、Change On Year(%) 列の後に Category のドロップをキャンセルできます。
+列のドラッグアンドドロップ操作をカスタマイズするための列移動に関連するイベントが複数あります。[`columnMovingStart`]({environment:angularApiUrl}/classes/igxgridcomponent.html#columnMovingStart)、[`columnMoving`]({environment:angularApiUrl}/classes/igxgridcomponent.html#columnmoving)、[`columnMovingEnd`]({environment:angularApiUrl}/classes/igxgridcomponent.html#columnMovingEnd) があります。
```
Comment (mddragnev) [inline jp/components/grids_templates/column-moving.md:121]: the redirect to columnMoving api docs should be changed from ...#columnmoving to ...#columnMoving

File: jp/components/grids_templates/keyboard-navigation.md [hunk 4]
```diff
@@ -150,29 +150,29 @@ _language: ja
...
 ```html
-<igx-hierarchical-grid #grid1 [data]="data" (onGridKeydown)="customKeydown($event, grid1)">
+<igx-hierarchical-grid #grid1 [data]="data" (gridKeydown)="customKeydown($event, grid1)">
     <igx-row-island [key]="'Albums'" (onGridCreated)="childGridCreated($event)">
     </igx-row-island>
 </igx-hierarchical-grid>
 ```
 
-igxHierarchicalGrid子グリッドにカスタムキーボードナビゲーションを追加するには、各子グリッドを [`onGridKeydown`]({environment:angularApiUrl}/classes/igxhierarchicalgridcomponent.html#ongridkeydown) イベントにサブスクライブする必要があります。そのため、上記の例で[`onGridCreated`]({environment:angularApiUrl}/classes/igxrowislandcomponent.html#ongridcreated) イベントのイベント ハンドラーを登録しました。
+igxHierarchicalGrid子グリッドにカスタムキーボードナビゲーションを追加するには、各子グリッドを [`gridKeydown`]({environment:angularApiUrl}/classes/igxhierarchicalgridcomponent.html#gridKeydown) イベントにサブスクライブする必要があります。そのため、上記の例で[`onGridCreated`]({environment:angularApiUrl}/classes/igxrowislandcomponent.html#ongridcreated) イベントのイベント ハンドラーを登録しました。
```
Comment (mddragnev) [inline jp/components/grids_templates/keyboard-navigation.md:170]: onGridCreated event should be gridCreated

File: en/components/grids_templates/keyboard-navigation.md [hunk 5]
```diff
@@ -150,29 +150,29 @@ Overriding the default behavior for a certain key or keys combination is one of
...
 ```html
-<igx-hierarchical-grid #grid1 [data]="data" (onGridKeydown)="customKeydown($event, grid1)">
+<igx-hierarchical-grid #grid1 [data]="data" (gridKeydown)="customKeydown($event, grid1)">
     <igx-row-island [key]="'Albums'" (onGridCreated)="childGridCreated($event)">
     </igx-row-island>
 </igx-hierarchical-grid>
 ```
 
-In order to add custom keyboard navigation to igxHierarchicalGrid child grids too, each child grid should subscribe to [`onGridKeydown`]({environment:angularApiUrl}/classes/igxhierarchicalgridcomponent.html#ongridkeydown) event. That's why in example above we have registered and event handler for for the [`onGridCreated`]({environment:angularApiUrl}/classes/igxrowislandcomponent.html#ongridcreated) event:
+In order to add custom keyboard navigation to igxHierarchicalGrid child grids too, each child grid should subscribe to [`gridKeydown`]({environment:angularApiUrl}/classes/igxhierarchicalgridcomponent.html#gridKeydown) event. That's why in example above we have registered and event handler for for the [`onGridCreated`]({environment:angularApiUrl}/classes/igxrowislandcomponent.html#ongridcreated) event:
```
Comment (mddragnev) [inline en/components/grids_templates/keyboard-navigation.md:170]: onGridCreated event should be gridCreated

File: kr/components/grids_templates/column-moving.md [hunk 6]
```diff
@@ -114,17 +114,17 @@ const idColumn = grid.getColumnByName("ID");
...
 
 
 #### 이벤트
 
-열의 드래그 앤드 드롭 조작을 설정하기 위해 열 이동과 관련된 복수의 이벤트가 제공됩니다. [`onColumnMovingStart`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#oncolumnmovingstart), [`onColumnMoving`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#oncolumnmoving) 및 [`onColumnMovingEnd`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#oncolumnmovingend)가 있습니다. 
-[`@@igSelector`]({environment:angularApiUrl}/classes/@@igTypeDoc.html)의 [`onColumnMovingEnd`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#oncolumnmovingend) 이벤트를 처리하고 열을 새로운 위치로 드롭할 때 일부 사용자 논리를 구현할 수 있습니다. 예를 들면, Change On Year(%) 열 뒤에 카테고리 드롭을 취소할 수 있습니다.
+열의 드래그 앤드 드롭 조작을 설정하기 위해 열 이동과 관련된 복수의 이벤트가 제공됩니다. [`columnMovingStart`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#columnMovingStart), [`columnMoving`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#columnmoving) 및 [`columnMovingEnd`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#columnMovingEnd)가 있습니다.
```
Comment (mddragnev) [inline kr/components/grids_templates/column-moving.md:122]: same as en and jp

File: kr/components/grids_templates/keyboard-navigation.md [hunk 7]
```diff
@@ -113,19 +113,19 @@ You can try the `actions below` in order to observe the custom keyboard navigati
 }
 
 @@if (igxName === 'IgxHierarchicalGrid') {
-In order to add custom keyboard navigation to igxHierarchicalGrid child grids [`onGridCreated`]({environment:angularApiUrl}/classes/igxrowislandcomponent.html#ongridcreated) should be handled and each child grid should subscribe to [`onGridKeydown`]({environment:angularApiUrl}/classes/igxhierarchicalgridcomponent.html#ongridkeydown) event.
+In order to add custom keyboard navigation to igxHierarchicalGrid child grids [`onGridCreated`]({environment:angularApiUrl}/classes/igxrowislandcomponent.html#ongridcreated) should be handled and each child grid should subscribe to [`gridKeydown`]({environment:angularApiUrl}/classes/igxhierarchicalgridcomponent.html#gridKeydown) event.
```
Comment (mddragnev) [inline kr/components/grids_templates/keyboard-navigation.md:116]: `onGridCreated` should be `gridCreated`

**Review by mddragnev — CHANGES_REQUESTED** [review]

File: en/components/grids_templates/row-drag.md [hunk 8]
```diff
@@ -663,7 +663,7 @@ We want to subscribe to the `dragMove` event only of the specific row we're drag
 class MyRowGhostComponent {
     ...
     public ngAfterViewInit(): void {
-        this.grid.onRowDragStart.pipe(takeUntil(this.destroy$)).subscribe(this.onRowDragStart.bind(this));
+        this.grid.rowDragStart.pipe(takeUntil(this.destroy$)).subscribe(this.onRowDragStart.bind(this));
```
Comment (mddragnev) [inline en/components/grids_templates/row-drag.md:666]: this is not fixed in the jp one

File: en/components/grids_templates/row-drag.md [hunk 9]
```diff
@@ -672,7 +672,7 @@ class MyRowGhostComponent {
         }
         const directive = e.dragDirective;
         directive.dragMove
-            .pipe(takeUntil(this.grid.onRowDragEnd))
```
Comment (mddragnev) [inline en/components/grids_templates/row-drag.md:675]: same as above

File: en/components/grids_templates/row-drag.md [hunk 10]
```diff
@@ -824,10 +824,10 @@ We create and subscribe to the `interval` when the pointer reaches the grid's ed
 ```typescript
 class MyGridScrollComponent {
     public ngAfterViewInit() {
-        this.grid.onRowDragStart
```
Comment (mddragnev) [inline en/components/grids_templates/row-drag.md:827]: not fixed in jp

File: en/components/grids_templates/keyboard-navigation.md [hunk 11]
```diff
@@ -150,29 +150,29 @@ Overriding the default behavior for a certain key or keys combination is one of
...
+<igx-grid #grid1 [data]="data" [primaryKey]="'ProductID'" (gridKeydown)="customKeydown($event)">
 ```
 }
 
 @@if (igxName === 'IgxHierarchicalGrid') {
 
 ```html
-<igx-hierarchical-grid #grid1 [data]="data" (onGridKeydown)="customKeydown($event, grid1)">
+<igx-hierarchical-grid #grid1 [data]="data" (gridKeydown)="customKeydown($event, grid1)">
     <igx-row-island [key]="'Albums'" (onGridCreated)="childGridCreated($event)">
```
Comment (mddragnev) [inline en/components/grids_templates/keyboard-navigation.md:165]: onGridCreated should be gridCreated

**Review by mddragnev — CHANGES_REQUESTED** [review]

**Comment by mddragnev** [comment]
@3phase, we actually have one more PR targeting common renaming : https://github.com/IgniteUI/igniteui-angular/pull/9278
As long as I saw the only change here should be `onGridCreated` -> `gridCreated`. Could you please look it up

**Review by mddragnev — DISMISSED** [review]

File: jp/components/general/update-guide.md [hunk 12]
```diff
@@ -367,7 +367,7 @@ grid.getRowByIndex(0).expanded = false;
 ```
 *row.rowData* および *row.rowID* は非推奨であり、バージョン 13 で完全に削除されます。代わりに *row.data* と *row.key* を使用してください。
 
-* *onRowPinning* によって発行されたイベント引数の *row* プロパティ、および *onRowDragStart* によって発行されたイベント引数の *dragData* プロパティ、*onRowDragEnd* は [`RowType`]({environment:angularApiUrl}/interfaces/rowtype.html) を実装しています。 
+* *rowPinning* によって発行されたイベント引数の *row* プロパティ、および *onRowDragStart* によって発行されたイベント引数の *dragData* プロパティ、*onRowDragEnd* は [`RowType`]({environment:angularApiUrl}/interfaces/rowtype.html) を実装しています。
```
Comment (ChronosSF) [inline jp/components/general/update-guide.md:370]: This is in a 11.1 to 12.0 upgrade guide. If you are sure the events were changed in 12.0, first this should be fixed in the EN and KR update guides. Also check if the `onRowDragEnd` & `onRowDragStart` in this same line should be changed.

Comment (3phase) [reply jp/components/general/update-guide.md]: @ChronosSF,
It's fixed now

**Review by ChronosSF — APPROVED** [review]

**Pull request merged**